pkg/formatters/summaryFormatter.go

```diff
@@ -42,14 +42,18 @@ func (*SummaryFormatter) FormatRuns(testResultsData []FormattableTest) (string,
 
 		table = append(table, headers)
 		for _, run := range testResultsData {
-			var line []string
-			submittedTime := run.QueuedTimeUTC
-			submittedTimeReadable := formatTimeReadable(submittedTime)
-
-			accumulateResults(resultCountsMap, run)
-
-			line = append(line, submittedTimeReadable, run.Name, run.Requestor, run.Status, run.Result, run.TestName)
-			table = append(table, line)
+			if !run.Lost {
```

Comment:
Can we swap this statement round so that the lost counter is in the if to make it easier to follow the code flow i.e. 
```
if run.lost{
  resultCountsMap[RUN_RESULT_LOST] += 1
} else {
```